coil-compose-core/src/commonMain/kotlin/coil3/compose/AsyncImagePainter.kt

```diff
@@ -385,15 +390,15 @@ class AsyncImagePainter internal constructor(
 
 @ReadOnlyComposable
 @Composable
-private fun previewHandler(): AsyncImagePreviewHandler? {
+internal fun previewHandler(): AsyncImagePreviewHandler? {
```

Comment:
Is there a benefit to making these internal?

Reply:
yes, I have to make it internal as the previous usage was from the same file from rememberAsyncImagePainter(). now I am creating AsyncImagePainter manually differently, and I invoke previewHandler() from AsyncImage composable (from a different file)